### PR TITLE
fix typos and changes for source's email templates

### PR DESCRIPTION
**Pull request by lpichler**
Subject: Source XXX has availability status change for _Application_

Body: _Application's_ availability status was changed for from **available** to **unavailable**.

For more information please check summary page of **MySource** source.

cc @josejulio 

~~please check especially conditions in `availabilityStatusEmailTitle.txt` - I was not sure how to check strings and whether I can  have nested conditions, thanks!~~

### CODE REVIEW

File: engine/src/main/resources/templates/Sources/availabilityStatusEmailBody.html [hunk 1]
```diff
@@ -32,13 +32,12 @@
...
-                        {action.context.resource_display_name}'s availability status was changed for  from <b>{action.context.resource_display_name}</b> to 
-                        <b>{action.context.resource_display_name}<b>.
-
+                        {action.context.resource_display_name}'s availability status was changed from <b>{action.context.previous_availability_status}</b> to 
+                        <b>{action.context.current_availability_status}</b>.
                     </p>
                     <p>
                         {#if action.context.source_id > 0}
-                        For more information please check summary page of <a class="rh-url" href="https://console.stage.redhat.com/settings/sources/detail/{action.context.source_id}"> {action.context.source_name} source</a>.
+                        For more information please check summary page of <a class="rh-url" href="https://console.redhat.com/settings/sources/detail/{action.context.source_id}"> {action.context.source_name} source</a>.
```
Comment (gwenneg) [inline engine/src/main/resources/templates/Sources/availabilityStatusEmailBody.html:40]: With the following change, the host will change dynamically depending on the environment where the template is rendered:
```suggestion
                        For more information please check summary page of <a class="rh-url" href="{environment.url('/settings/sources/detail/')}{action.context.source_id}"> {action.context.source_name} source</a>.
```

**Comment by lpichler** [comment]
@gwenneg I added your suggestion, you can re-review, thanks!

**Review by gwenneg — APPROVED** [review]
LGTM

**Comment by gwenneg** [comment]
Do you want to test that on stage today? I can deploy the modified template there if needed.

**Comment by lpichler** [comment]
@gwenneg yes that would be great!

**Comment by gwenneg** [comment]
Ok, deployment in progress... 😃

**Comment by gwenneg** [comment]
The updated template should be available on stage now.

**Comment by lpichler** [comment]
> The updated template should be available on stage now.

I am confirming it, I just tested it 👍

**Pull request merged**